study1/week5/group-anagrams.java

```diff
@@ -0,0 +1,41 @@
+// https://leetcode.com/problems/group-anagrams/
+// Time Complexity: O(nlogn), n: length of string which is the longest
```

Comment:
`nlogn` is only for a single word, but we have `strs.length` words, so can you rethink complexity?